post-processor/yandex-import/post-processor.go

```diff
@@ -42,6 +50,14 @@ type Config struct {
 	// leave it in the bucket, `false` to remove it. Default is `false`.
 	SkipClean bool `mapstructure:"skip_clean" required:"false"`
 
+	// StorageEndpoint custom Yandex Object Storage endpoint to upload image, Default `storage.yandexcloud.net`.
+	StorageEndpoint string `mapstructure:"storage_endpoint" required:"false"`
+	// StorageEndpointAutoresolve auto resolve storage endpoint via YC Public API ListEndpoints call. Option has
+	// precedence over 'storage_endpoint' option.
```

Comment:
If we reject using both at the same time through a check in `Configure` (which I think would make sense), this comment should be amended to state that this and the `storage_endpoint` options are mutually exclusive.